bitrise.yml

```diff
@@ -143,6 +143,7 @@ workflows:
       - run_linter
       - run_unit_tests
       - run_integration_tests
+      - run_e2e_s8s_upload
       - run_smoke_tests
```

Comment:
change request/ If we want to run it after merge to develop, this should be started from `push_to_develop_or_master`, not the generic `run_conditioned_workflows`. With existing setup it will be ran after each push to feature branch, no?

Reply:
Good catch!